drivers/net/eth_rtt.c

```diff
@@ -544,5 +544,5 @@ static const struct ethernet_api if_api = {
 
 /** Initialization of network device driver. */
 ETH_NET_DEVICE_INIT(eth_rtt, CONFIG_ETH_RTT_DRV_NAME, eth_rtt_init,
-		    &context_data, NULL, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT,
-		    &if_api, CONFIG_ETH_RTT_MTU);
+	device_pm_control_nop, &context_data, NULL,
```

Comment:
Why do you change the alignment?

Reply:
My editor does that when I break a line.
I'll change it back.